src/shared/filesystem.ts

```diff
@@ -15,8 +15,12 @@ export interface Stats extends fs.Stats {
     // fs.Stats is a class, so for easy mocking we code against an interface with the same shape.
 }
 
+export interface BigIntStats extends fs.BigIntStats {
+    // fs.BigIntStats is a class, so for easy mocking we code against an interface with the same shape.
+}
```

Comment:
Doesn't look like we're using this anywhere in this PR, unless I'm missing something?

Reply:
This will be fixed by the fs changes but this is to get it to build